mcp-server/README.md

```
@@ -0,0 +1,129 @@
+pwd  # Ez megmondja a teljes útvonalat
+```
+
+### 3. Indítsd újra Claude Desktop-ot
+
+Az MCP szerver most elérhető lesz Claude számára a beszélgetésekben.
+
+## Használat példa beszélgetésben
+
+Miután beállítottad az MCP szervert:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)